api/applications/applications.proto

```diff
@@ -38,10 +38,22 @@ service Applications {
     }
 }
 
+// This object represents a single condition for a Kubernetes object.
+// It roughly matches the Kubernetes type defined here: https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Condition
+message Condition {
+    string type      = 1;
+    string status    = 2;
+    string reason    = 3;
+    string message   = 4;
+    string timestamp = 5;
```

Comment:
Can you elaborate on using string as the type instead of timestamp?  Having timestamp instead of strings makes it easier for non-human interfaces to consume. Also, a UI will have an easier time converting to local time when displaying to the user.

Reply:
Looks like I used an `int32` in the Flux UI:
https://github.com/fluxcd/webui/blob/f48d23e5095a04949ad276fc6da1349f380df457/pkg/clustersserver/clustersserver.go#L712

Which comes back in seconds. Hopefully enough resolution?:
https://github.com/fluxcd/webui/blob/f48d23e5095a04949ad276fc6da1349f380df457/ui/pages/Events.tsx#L34

Will fix!

Reply:
Changed to `int32`